GVFS/GVFS.Common/VersionResponse.cs

```diff
@@ -0,0 +1,15 @@
+using Newtonsoft.Json;
+
+namespace GVFS.Common
+{
+    public class VersionResponse
+    {
+        public string Version { get; set; }
+
+        public static VersionResponse FromJsonString(string jsonString)
+        {
+            JsonSerializer serializer = new JsonSerializer();
+            return JsonConvert.DeserializeObject<VersionResponse>(jsonString);
```

Comment:
`serializer` isn't used and I would remove this method and just call the `JsonConvert.DeserializeObject<VersionResponse>` inline.